src/stats.c

```diff
@@ -230,44 +283,36 @@ stats_arena_bins_print(emitter_t *emitter, bool mutex, unsigned i) {
 
 	emitter_row_t row;
 	emitter_row_init(&row);
-#define COL(name, left_or_right, col_width, etype)			\
-	emitter_col_t col_##name;					\
-	emitter_col_init(&col_##name, &row);				\
-	col_##name.justify = emitter_justify_##left_or_right;		\
-	col_##name.width = col_width;					\
-	col_##name.type = emitter_type_##etype;				\
-	emitter_col_t header_col_##name;				\
-	emitter_col_init(&header_col_##name, &header_row);		\
-	header_col_##name.justify = emitter_justify_##left_or_right;	\
-	header_col_##name.width = col_width;				\
-	header_col_##name.type = emitter_type_title;			\
-	header_col_##name.str_val = #name;
-
-	COL(size, right, 20, size)
-	COL(ind, right, 4, unsigned)
-	COL(allocated, right, 13, uint64)
-	COL(nmalloc, right, 13, uint64)
-	COL(ndalloc, right, 13, uint64)
-	COL(nrequests, right, 13, uint64)
-	COL(nshards, right, 9, unsigned)
-	COL(curregs, right, 13, size)
-	COL(curslabs, right, 13, size)
-	COL(regs, right, 5, unsigned)
-	COL(pgs, right, 4, size)
+
+	COL_HDR(row, size, right, 20, size)
+	COL_HDR(row, ind, right, 4, unsigned)
+	COL_HDR(row, allocated, right, 13, uint64)
+	COL_HDR(row, nmalloc, right, 13, uint64)
+	COL_HDR(row, nmalloc_ps, right, 13, uint64)
+	COL_HDR(row, ndalloc, right, 13, uint64)
+	COL_HDR(row, ndalloc_ps, right, 13, uint64)
+	COL_HDR(row, nrequests, right, 13, uint64)
+	COL_HDR(row, nrequests_ps, right, 13, uint64)
+	COL_HDR(row, nshards, right, 9, unsigned)
+	COL_HDR(row, curregs, right, 13, size)
+	COL_HDR(row, curslabs, right, 13, size)
+	COL_HDR(row, regs, right, 5, unsigned)
+	COL_HDR(row, pgs, right, 4, size)
 	/* To buffer a right- and left-justified column. */
-	COL(justify_spacer, right, 1, title)
-	COL(util, right, 6, title)
-	COL(nfills, right, 13, uint64)
-	COL(nflushes, right, 13, uint64)
-	COL(nslabs, right, 13, uint64)
-	COL(nreslabs, right, 13, uint64)
-#undef COL
+	COL_HDR(row, justify_spacer, right, 1, title)
+	COL_HDR(row, util, right, 6, title)
+	COL_HDR(row, nfills, right, 13, uint64)
+	COL_HDR(row, nfills_ps, right, 13, uint64)
+	COL_HDR(row, nflushes, right, 13, uint64)
+	COL_HDR(row, nflushes_ps, right, 13, uint64)
+	COL_HDR(row, nslabs, right, 13, uint64)
+	COL_HDR(row, nreslabs, right, 13, uint64)
+	COL_HDR(row, nreslabs_ps, right, 13, uint64)
```

Comment:
Please also try tightening the spacing after using the `#/sec` for header. The stats looks pretty messy when line wraps around.